languages/js/src/Polar.ts

```diff
@@ -183,77 +167,89 @@ export class Polar {
   }
 
   /**
-   * Start a REPL session.
+   * Register a JavaScript class for use in Polar policies.
    */
-  async repl(files?: string[]): Promise<void> {
-    const rl = createInterface({
-      input: process.stdin,
-      output: process.stdout,
-      prompt: FG_BLUE + 'query> ' + RESET,
-      tabSize: 4,
-    });
+  registerClass<T>(cls: Class<T>, alias?: string): void {
+    const name = this.#host.cacheClass(cls, alias);
+    this.registerConstant(name, cls);
+  }
 
+  /**
+   * Register a JavaScript value for use in Polar policies.
+   */
+  registerConstant(name: string, value: any): void {
+    const term = this.#host.toPolar(value);
+    this.#ffiPolar.registerConstant(name, JSON.stringify(term));
+  }
+
+  /** Start a REPL session. */
+  async repl(files?: string[]): Promise<void> {
     let loadError;
     try {
       if (files?.length) await Promise.all(files.map(f => this.loadFile(f)));
     } catch (e) {
       loadError = e;
     }
-    if (loadError !== undefined) {
-      console.error(FG_RED + 'One or more files failed to load.' + RESET);
-      console.error(loadError.message);
+    if (loadError !== undefined) printError(loadError);
```

Comment:
Why not just print these error at 191?

Reply:
I think this is a holdover from... something. But I can't remember why I did it this way. Removed in 6207cb6